src/main/java/org/sonatype/repository/conan/internal/metadata/ConanCoords.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2017-present Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+package org.sonatype.repository.conan.internal.metadata;
+
+public class ConanCoords
+{
+  private String group;
```

Comment:
I assume all these instance fields could be `final`? (I see no way they could be mutated given that this seems to be going for an immutable data type anyway, so why not make it explicit in the code?)

Reply:
https://github.com/sonatype-nexus-community/nexus-repository-conan/pull/15/commits/539b6b82c10176e161748f2efbdc20236bde1fa6